### PR TITLE
Fixed empty message bug

### PR DESCRIPTION
**Pull request by C-Anirudh**
Fixes: #67 
Changes made:

- Empty messages are no longer allowed.

- `Type a message` was made placeholder text instead of plain text in input field.

- Removed alert for trying to send empty messages as it is cumbersome and doesn't look professional.

[video of chat screen after changes](https://drive.google.com/file/d/1yYXbGUpyRikf5mTkmEvMfXZaJG4DnER6/view)

### CODE REVIEW

File: view/app-jarvis.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ app.controller('MainController', function($scope,$location,$rootScope,$http) {
 	});
 
 	$scope.addMessagesToStack = function() {
-		if (!$scope.message.startsWith('Type a message')) {
+		if (!$scope.message == '') {
```
Comment (Harkishen-Singh) [inline view/app-jarvis.js]: either use **===** or better try with ```if($scope.message.length)```

Comment (C-Anirudh) [reply view/app-jarvis.js]: @Harkishen-Singh  thanks for the tip.

**Comment by Harkishen-Singh** [comment]
please attach a video whenever requesting a UI update!

**Comment by C-Anirudh** [comment]
> please attach a video whenever requesting a UI update!

Added link to video in PR comment.

File: view/components/main.html [hunk 2]
```diff
@@ -38,79 +16,101 @@
...
+                <input type="text" name="" id="message-input" data-ng-model="message" placeholder="Type a message">
+                <button id="message-bar-send" class="btn btn-danger" data-ng-click="addMessagesToStack()">
+                    Send
+                </button>
+                <!-- banner -->
+                <div id="banner-listen" data-ng-show="showLoaderListening">
+                    <span>Listening ... </span>
+                    <img src="../assets/icons/load.gif" alt="listening to you. Please keep speaking.." id="loader" srcset="">
+                </div>
+                <img src="../assets/icons/mic.png" alt="Click to Start/Stop speaking" id="button"
```
Comment (Harkishen-Singh) [inline view/components/main.html]: could you please confirm if this functionality is working in the newer changes you have pushed?

Comment (C-Anirudh) [reply view/components/main.html]: The button works, the gif loads when I click the button and vanishes when I click on it again.

**Pull request merged**